src/BitStringAddresses/bosefs.jl

```diff
@@ -149,24 +156,19 @@ a total of `N` particles.
 # Examples
 ```jldoctest
 julia> near_uniform(BoseFS{7,5})
-BoseFS{7,5}((2, 2, 1, 1, 1))
+BoseFS{7,5}(2, 2, 1, 1, 1)
 
 julia> near_uniform(FermiFS{3,5})
-FermiFS{3,5}((1, 1, 1, 0, 0))
+FermiFS{3,5}(1, 1, 1, 0, 0)
 ```
 """
 function near_uniform(::Type{<:BoseFS{N,M}}) where {N,M}
     return BoseFS{N,M}(near_uniform_onr(Val(N),Val(M)))
 end
 near_uniform(b::AbstractFockAddress) = near_uniform(typeof(b))
 
-"""
-    onr(bs)
-
-Compute and return the occupation number representation of the bit string
-address `bs` as an `SVector{M,Int32}`, where `M` is the number of modes.
-"""
 onr(b::BoseFS{<:Any,M}) where {M} = to_bose_onr(b.bs, Val(M))
+const occupation_number_representation = onr # resides here because `onr` has to be defined
```

Comment:
Well, I was thinking to leave it around because it is so convenient. Maybe work towards using the full form in the tutorials, etc., in order order to not set a bad example for naming conventions 😄